sympy/simplify/simplify.py

```diff
@@ -33,6 +33,42 @@
 import mpmath
 
 
+def separable(expr, symbols=[]):
+    """Return the subset of symbols which appear in expr that
+    test as being positively separable from other symbols in expr
+    and those that do not test as being unseparable.
```

Comment:
This needs better wording. I don't understand what the second set is...

Reply:
okkk

Reply:
what about this-
```
Return the subset of symbols which appear in expr that
    test as being positively separable from other symbols in expr and those that do not classify as unseparable
.
```

Reply:
The double negative in "not unseparable" is confusing. What are you trying to say?

Reply:
the second one is set of that element which is not able to classify as separable but also they are not classified as unseparable so this set of the element can be separable or unseparable.

Reply:
If I wanted to describe that in the docstring I would say:

Return the set of symbols that are separable and the set of symbols that cannot be determined as either separable or unseparable.

However, why is that a useful thing to return? We need to go back to what the purpose of this function is...

Reply:
> However, why is that a useful thing to return? We need to go back to what the purpose of this function is...

If the separable function only returns set of separated vars then may be a user will be in confusion that other vars are not separable.
other think is - is_constant function is not always able to tell so for those it returns None.
So I think we should return both sets.